### DISCUSSION TITLE
Is there a pre built libtree-sitter?

### DISCUSSION THREAD

**Discussion by realhackcraft** [tree-sitter/tree-sitter#3894] [Q&A] [answered]
Is there? If not, is it possible to just compile the lib without the cli?

**Comment by sogaiu** [comment]
Regarding compiling the library, may be [these lines](https://github.com/tree-sitter/tree-sitter/blob/0f15779863131bbe9e8ae2e4d09348e9e5e70542/.github/workflows/build.yml#L155-L179) give some hints.

**Comment by amaanq** [comment]
just run `make`.

**Reply by sogaiu** [reply, marked as answer]
I think that won't work for some Windows setups.

For those Windows setups where `make` invocation doesn't succeed, I think it's possible the cmake approach (listed in some of the lines linked above) might work.
